src/pages/filter/dtree/components/modals/components/compound-het-dialog/compound-het-dialog.tsx

```diff
@@ -49,16 +48,17 @@ export const ModalCompoundHet = observer((): ReactElement => {
       param,
       mode,
     })
-    modalsVisibilityStore.closeModalCompoundHet()
+    modalsVisibilityStore.closeCompoundHetDialog()
   }, [])
 
   return (
-    <ModalBase minHeight={340}>
-      <HeaderModal
-        groupName={attributeName}
-        handleClose={modalsVisibilityStore.closeModalCompoundHet}
-      />
-
+    <Dialog
+      isOpen={modalsVisibilityStore.isCompoundHetDialogVisible}
+      onClose={modalsVisibilityStore.closeCompoundHetDialog}
+      title={attributeName}
+      width="m"
+      actions={''}
```

Comment:
That's some weird stuff here. Probably it's better to add a 'hideActions' prop to 'DialogCard'